### PR TITLE
reduce work in polyfill by doing server side lang detection

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
@cadecairos this should fix it if my hypothesis is correct. R?

Basically, we use server side lang detection now for polyfill.

**Comment by ScottDowne** [comment]
Shoot, issue, it's `*only*` using lang headers server side, it ignores the url path, which it should use first.

File: server.js [hunk 1]
```diff
@@ -244,7 +249,12 @@ module.exports = function(options) {
       method: 'GET',
       path: '/api/polyfill.js',
       handler: function(request, reply) {
-        var features = request.query.features;
+
+        var header = Parser.parse(request.headers["accept-language"]);
```
Comment (cadecairos) [inline server.js]: what happens if accept-language isn't defined?

**Comment by ScottDowne** [comment]
@cadecairos should be ready for review, and I'll want to test this for memory usage, if it fixes it, somehow.

**Comment by cadecairos** [comment]
R+

**Pull request merged**